### PR TITLE
front: remove trailing slashes in osrdEditoastApi URLs

### PR DESCRIPTION
**Pull request by leovalais**
Refs https://github.com/OpenRailAssociation/osrd/pull/8047#pullrequestreview-2199001346

### CODE REVIEW

**Comment by emersion** [comment]
I don't think we even need to set the `url` field here, it's already in `generatedEditoastApi` and we don't need to override it. Same for `method`.

**Comment by emersion** [comment]
Hm, my bad, it's a callback so ~~probably~~ [definitely](https://github.com/reduxjs/redux-toolkit/blob/75b4f698e1c58efa6b19096abe5850da0a431da0/packages/toolkit/src/query/createApi.ts#L329) won't work…

File: front/src/common/api/osrdEditoastApi.ts [hunk 1]
```diff
@@ -34,7 +34,7 @@ const osrdEditoastApi = generatedEditoastApi.enhanceEndpoints({
     },
     deleteV2TrainSchedule: {
       query: (queryArg) => ({
-        url: `/v2/train_schedule/`,
+        url: `/v2/train_schedule`,
         method: 'DELETE',
         body: queryArg.body,
       }),
```
Comment (emersion) [inline front/src/common/api/osrdEditoastApi.ts]: Upon further inspection, it seems like `query` is identical to the one in `generatedEditoastApi` so can just be dropped altogether.

Same for `postV2TimetableByIdTrainSchedule`.

**Review by emersion — APPROVED** [review]
LGTM!

**Review by SharglutDev — APPROVED** [review]
Lgtm

**Pull request merged**